diff --git a/src/NativeAsyncStorageModule.ts b/src/NativeAsyncStorageModule.ts
index be7e05537f5aa0a8ca7a199dc3cb6a2b1a7a8c28..51ca3d2c8f4de31b9b59653e783fff8daf35aadb 100644
--- a/src/NativeAsyncStorageModule.ts
+++ b/src/NativeAsyncStorageModule.ts
@@ -1,28 +1,27 @@
 import type { TurboModule } from "react-native";
 import { TurboModuleRegistry } from "react-native";
-import type { ErrorLike } from "./types";
 
 export interface Spec extends TurboModule {
   multiGet: (
     keys: string[],
-    callback: (error?: ErrorLike[], result?: [string, string][]) => void
+    callback: (error?: Object[], result?: Object[]) => void
   ) => void;
   multiSet: (
-    kvPairs: [string, string][],
-    callback: (error?: ErrorLike[]) => void
+    kvPairs: Object[],
+    callback: (error?: Object[]) => void
   ) => void;
   multiRemove: (
-    keys: readonly string[],
-    callback: (error?: ErrorLike[]) => void
+    keys: string[],
+    callback: (error?: Object[]) => void
   ) => void;
   multiMerge: (
-    kvPairs: [string, string][],
-    callback: (error?: ErrorLike[]) => void
+    kvPairs: Object[],
+    callback: (error?: Object[]) => void
   ) => void;
   getAllKeys: (
-    callback: (error?: ErrorLike[], result?: [string, string][]) => void
+    callback: (error?: Object[], result?: string[]) => void
   ) => void;
-  clear: (callback: (error?: ErrorLike[]) => void) => void;
+  clear: (callback: (error?: Object[]) => void) => void;
 }
 
 export default TurboModuleRegistry.get<Spec>("RNCAsyncStorage");
